apps/tenant-management-webapp-e2e/src/integration/service-status.feature

```diff
@@ -289,8 +289,29 @@ Feature: Service status
     Then no critical or serious accessibility issues on "status applications page"
     When the user selects "Notices" tab for "Status"
     Then no critical or serious accessibility issues on "status notices page"
-## CS-1863 is pending for fix
-# When the user clicks Add notice button
-# Then the user views Add notice dialog
-# And no critical or serious accessibility issues for "add notice modal" on "status notices page"
-# When the user clicks Cancel button in notice modal
\ No newline at end of file
+  ## CS-1863 is pending for fix
+  # When the user clicks Add notice button
+  # Then the user views Add notice dialog
+  # And no critical or serious accessibility issues for "add notice modal" on "status notices page"
+  # When the user clicks Cancel button in notice modal
+
+  @TEST_CS-2249 @REQ_CS-2058 @regression
+  Scenario: As a tenant admin, I can set a status application to be monitoring only, so it does not appear on the public status page
+    Given a tenant admin user is on status applications page
+    # Set monitor only for an application
+    When the user "selects" Monitor only checkbox for "autotest-testMonitoring"
```

Comment:
I can not find "selects" Monitor checkbox, which I don't see in status application. Is it on Status service > Applications? or https://status.adsp-uat.alberta.ca/autotest

Reply:
It's in service-status in tenant management app. Status app is the public facing app for public to see the status of the applications.

Reply:
Than is was I thought, in this case it should be status service>applications tab

Reply:
What tab? There is no step involving tabs. What's wrong with the following?

    Given a tenant admin user is on status applications page
    # Set monitor only for an application
    When the user "selects" Monitor only checkbox for "autotest-testMonitoring"